docs/content/en/docs/concepts/architecture/keptn-apps/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: KeptnApp and KeptnWorkload resources
+linktitle: Keptn Applications and Keptn Workloads
+description: How Keptn applications work
+weight: 50
+cascade:
+---
+
+## Keptn Workloads
+
+A
+[KeptnWorkload](../../../crd-ref/lifecycle/v1alpha3/#keptnworkload)
+resource augments a Kubernetes
+[Workload](https://kubernetes.io/docs/concepts/workloads/)
+with the ability to handle extra phases.
+It can execute the pre- and post-deployment checks of a Workload
+and run pre- and post-deployment tasks.
```

Comment:
To avoid confusion, I would suggest to always use the terms `evaluation` and `tasks`, instead of `check`,  to establish a clear relation to their respective custom resources.
```suggestion
It can execute the pre- and post-deployment evaluations of a Workload
and run pre- and post-deployment tasks.
```